dashboard/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
@christianvuerings Should I be sticking to an ECL-2.0 license, or can I use some other license as well. What would be the requirements for the license? Specifically, I was considering the GPL v3.0 license.